source/index.html.md

```diff
@@ -394,8 +394,10 @@ payment method; subsequent-in-series authorizations, however,
 must be made using the `card` payment method using the card details of the
 referenced previous-in-series authorization.
 
-Any first-in-series authorization must be made with strong customer
-authentication (SCA) regardless of the authorization amount.
+A card is in scope for SCA (strong customer authentication) if card country is
+in EEA. (Exceptions apply for UK.)
```

Comment:
I know there's a context, but this sounds absolute and is only true because we only have EU merchants.

```suggestion
With Clearhaus, a card is in scope for SCA (strong customer authentication) if card country is
in EEA. (Exceptions apply for UK.)
```

Reply:
But you are welcome to ignore me.